R/infix_spaces_linter.R

```diff
@@ -119,7 +118,8 @@ infix_spaces_linter <- function(exclude_operators = NULL, allow_multiple_spaces
           ]
         ]
       )
-    ]")
+    ")
+    xpath <- paste(glue::glue("//{infix_tokens}[{xp_condition}]"), collapse = " | ")
```

Comment:
this needs a different benchmark IMO, we didn't measure with vs. without conditions on each node

Reply:
good thinking. I'll cook up a benchmark based on default-linter-testcode.R